### PR TITLE
Reviewed PR-386

### PR DESCRIPTION
**Pull request by dg-latacora**
Updated to fix where [PR-386](https://github.com/crypto101/book/pull/386) changes crypto meanings

### CODE REVIEW

File: src/stream-ciphers.rst [hunk 1]
```diff
@@ -137,43 +137,42 @@ significantly fewer pixels per block than the larger block sizes in the
...
 
    C = ECB(E_k, A \| S)
 
 The secret suffix :math:`S` is specific to this system. The attacker's
-goal is to decrypt it. We'll see that being able to encrypt other
-messages surprisingly allows the attacker to decrypt the suffix. This
-:term:`oracle` might seem artificial, but is quite common in practice. A simple
-example would be a cookie encrypted with ECB, where the prefix :math:`A`
-is a name or an e-mail address field, controlled by the attacker.
+goal is to decrypt it. Ability to encrypt other
```
Comment (lvh) [inline src/stream-ciphers.rst:171]: This feels like it's missing a `The`?

File: src/stream-ciphers.rst [hunk 2]
```diff
@@ -137,43 +137,42 @@ significantly fewer pixels per block than the larger block sizes in the
...
+messages surprisingly allows the attacker to decrypt the suffix. The
```
Comment (lvh) [inline src/stream-ciphers.rst:172]: This should really be `this`: there are lots of kinds of oracles, this _specific_ one may seem artificial

File: src/stream-ciphers.rst [hunk 3]
```diff
@@ -137,43 +137,42 @@ significantly fewer pixels per block than the larger block sizes in the
...
 
-When we look at a picture of what would happen with an idealized
-encryption scheme, we notice that it looks like random noise. Keep in
-mind that “looking like random noise” doesn't mean something is properly
-encrypted: it just means that we can't inspect it using methods this
-trivial.
+Notice that an idealized
+encryption scheme looks like random noise.
+“Looking like random noise” does not mean something is properly
+encrypted: it just means that we cannot inspect it using trivial methods.
```
Comment (lvh) [inline src/stream-ciphers.rst:143]: For all these paragraphs: please consider M-q / filling these paragraphs to their previous consistent line length :)

**Review by lvh — CHANGES_REQUESTED** [review]
Looks great modulo handful of comments, thanks for working on this!

File: src/stream-ciphers.rst (hunk 1 above)
Comment (dg-latacora) [reply src/stream-ciphers.rst]: Added `The`

File: src/stream-ciphers.rst (hunk 3 above)
Comment (dg-latacora) [reply src/stream-ciphers.rst]: I ran M-q on above paragraph (line 140 - 143)

When you said "these paragraphs", do you mean M-q on all other paragraphs as well?

File: src/stream-ciphers.rst (hunk 2 above)
Comment (dg-latacora) [reply src/stream-ciphers.rst]: Got it. That makes sense

**Review by dg-latacora — COMMENTED** [review]
@lvh ,
I handled all except a question on https://github.com/crypto101/book/pull/410#discussion_r563381692

File: src/stream-ciphers.rst (hunk 3 above)
Comment (lvh) [reply src/stream-ciphers.rst]: Nah just the ones changed in this PR. Thanks!

**Review by lvh — APPROVED** [review]

**Pull request merged**